File: bin/Debug/AI.py
def f():
	for x in range(3):
		for y in range(3):
			if game.GetBox(x, y) == ClearBox:
				game.MakeMove(x, y) 
				return
def g():
	for x in range(1,3):
		for y in range(3):
			if game.GetBox(x, y) == ClearBox:
				game.MakeMove(x, y) 
				return
if player == "X":
	f()
else:
	g()

File: AI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using IronPython.Hosting;
using Microsoft.Scripting;
using Microsoft.Scripting.Hosting;

namespace TickTackToe
{
    class AI
    {
        private ScriptEngine m_engine = Python.CreateEngine();
        private ScriptScope m_scope = null;
        ScriptSource source;

        public void Init(string player)
        {
            m_scope = m_engine.CreateScope();
            m_scope.SetVariable("player", player);
            m_scope.SetVariable("ClearBox", BoxState.Clear);
            m_scope.SetVariable("XBox", BoxState.X);
            m_scope.SetVariable("OBox", BoxState.O);
            source = m_engine.CreateScriptSourceFromString(System.IO.File.ReadAllText("AI.py"), SourceCodeKind.AutoDetect);
           
        }

        public void MakeMove(GameControl gc)
        {
            m_scope.SetVariable("game", gc);          
            source.Execute(m_scope);
        }
    }
}
